db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.6.35, for osx10.11 (x86_64)
```

Comment:
This change should not be part of this commit. Once https://github.com/GSA/search-gov/pull/172 is merged, this PR can be rebased against master so that it doesn't include the DB change.

Reply:
reindex changes can not be done on one branch. We need to first deploy the migrations. Do the migration on production have it ready then deploy the second part.  When the migration pr goes to production, I will merge the master branch and you will not see the those changes on this branch anymore.